sys/shell/cmds/gnrc_icmpv6_echo.c

```diff
@@ -207,7 +208,13 @@ static int _configure(int argc, char **argv, _ping_data_t *data)
                     /* intentionally falls through */
                 case 's':
                     if ((++i) < argc) {
-                        data->datalen = atoi(argv[i]);
+                        value = atoi(argv[i]);
+
+                        if ((value < 0) || ((unsigned)value > (UINT16_MAX - sizeof(icmpv6_hdr_t)))) {
+                            printf("ICMPv6 datagram size should be in range 0-65527.\n");
```

Comment:
If you make this less verbose it should also still fit on those boards

Reply:
See #19933

Reply:
I reduce ROM usage for 20 bytes - now should compile without problems. Moreover I revert changes from PR #19933. Could we check what bors things about my changes? ;)